testnets/kima/assetlist.json

```diff
@@ -0,0 +1,34 @@
+{
+  "$schema": "../../assetlist.schema.json",
+  "chain_name": "kimatestnet",
+  "assets": [
+    {
+      "description": "The native token of Kima Chain",
+      "denom_units": [
+        {
+          "denom": "uKIMA",
+          "exponent": 0
+        },
+        {
+          "denom": "KIMA",
+          "exponent": 6
+        }
+      ],
+      "base": "uKIMA",
+      "name": "KIMA",
+      "display": "KIMA",
+      "symbol": "KIMA",
+      "logo_URIs": {
+        "png": "https://raw.githubusercontent.com/cosmos/chain-registry/master/kichain/images/uKIMA.png",
+        "svg": "https://raw.githubusercontent.com/cosmos/chain-registry/master/kichain/images/uKIMA.svg"
+      },
+      "coingecko_id": "kima",
+      "images": [
+        {
+          "png": "https://raw.githubusercontent.com/cosmos/chain-registry/master/kimachain/images/uKIMA.png",
+          "svg": "https://raw.githubusercontent.com/cosmos/chain-registry/master/kimachain/images/uKIMA.svg"
```

Comment:
Please upload these images (you might need to put them into /testnets/kimatestnet/images/ first, and then may move them over to /kima/images/ once the mainnet is ready to be registered.)